stories/Molecules/Blocks/StatsPanel/statspanel.scss

```diff
@@ -27,16 +27,27 @@
 
   &::after {
     @extend %absolute__content;
-
-    background:
-      linear-gradient(0deg,
-      $color-yellow 1.21%,
-      $color-yellow 56.78%,
-      rgba(255, 235, 0, 0.7) 100%);
+    
     opacity: 0;
     transition: all 0.4s;
   }
 
+  &.yellow::after {
+    background: linear-gradient(0deg, $color-yellow 1.21%, $color-yellow 56.78%, rgba(255, 235, 0, 0.7) 100%);
```

Comment:
@Shubham-Sharma-77 Can you create a global color list and use mixin here. Something like: 
declaring global var in scss something like: 
```
$yellow-gradient: $color-yellow 1.21%, $color-yellow 56.78%, rgba(255, 235, 0, 0.7) 100%;
$red-gradient: $color-red 1.21%, $color-red 56.78%, rgb(240 49 49 / 70%) 100%;
$green-gradient: $color-green 1.21%, $color-green 56.78%, rgb(83 230 80 / 70%) 100%;
$blue-gradient: $color-azure 1.21%, $color-azure 56.78%, rgb(101 204 244 / 70%) 100%;
```
and declaring a global mixin in scss something like:
```
@mixin gradient($direction, $list) { 
  background: -webkit-linear-gradient($direction, $list);
  background: -moz-linear-gradient($direction, $list);
  background: -o-linear-gradient($direction, $list);
  background: linear-gradient($direction, $list);  
}
```
and use it in scss classes like:
```
&.yellow::after {
  @include gradient(0deg, $yellow-gradient);
}
&.red::after {
  @include gradient(0deg, $red-gradient);
}
&.green::after {
  @include gradient(0deg, $green-gradient);
}
&.blue::after {
  @include gradient(0deg, $blue-gradient);
}
```